### ISSUE TITLE
VV fails to activate on matrixes

### ISSUE THREAD

**Issue by Pockets-byte** [unitystation/unitystation#6779]
Linux-staging

On staging if you try to activate VV on matrixes, it completely fails, leaving you with what was last activated
this is kinda a pain for testing various things, as well as controlling speeds on shuttles and other various things

**Comment by Pockets-byte** [comment]
If I click into the shuttle matrix move from a shuttle computer, it crashes my VV

**Comment by Bod9001** [comment]
"On staging if you try to activate VV on matrixes, it completely fails, leaving you with what was last activated
this is kinda a pain for testing various things, as well as controlling speeds on shuttles and other various things"

do you mean you verbals on matrices. The last option in the admin right click?

**Comment by Bod9001** [comment]
couldn't replicate the second one on the editor,

**Comment by Pockets-byte** [comment]
uhh?
the VV option in the right click menu acts as if it wasn't clicked

**Comment by pyrosshade** [comment]
have you tried testing this again? this seems like something that i fixed with #6664

**Comment by Pockets-byte** [comment]
testing it again now

**Comment by Pockets-byte** [comment]
I can confirm this still doesn't work on staging

**Comment by Pockets-byte** [comment]
https://user-images.githubusercontent.com/55299415/121761401-83b2c580-cafd-11eb-973d-165faf7e66b4.mov

I am clicking in this

**Comment by pyrosshade** [comment]
what shows in f5 when you try to do so?

**Comment by corp-0** [comment]
yeah, I also just verified it in the same session that video was captured. 
```
NullReferenceException: Object reference not set to an instance of an object
SweetExtensions.NetId (UnityEngine.GameObject go) (at <4029ab2637b6496a9cf4fd9ebd32c4f4>:0)
Messages.Client.VariableViewer.RequestBookshelfNetMessage.Send (UnityEngine.GameObject _TheObjectToView, System.String adminId, System.String adminToken) (at <4029ab2637b6496a9cf4fd9ebd32c4f4>:0)
RightClickManager+<>c__DisplayClass21_0.<Generate>b__0 () (at <4029ab2637b6496a9cf4fd9ebd32c4f4>:0)
UI.Core.RightClick.RightClickMenuController.OnClickAction (UnityEngine.EventSystems.PointerEventData eventData, UI.Core.RightClick.RightClickRadialButton button) (at <4029ab2637b6496a9cf4fd9ebd32c4f4>:0)
UnityEngine.Events.InvokableCall`2[T1,T2].Invoke (T1 args0, T2 args1) (at <e0a51238dca14258b8540510b8e6edba>:0)
UnityEngine.Events.UnityEvent`2[T0,T1].Invoke (T0 arg0, T1 arg1) (at <e0a51238dca14258b8540510b8e6edba>:0)
UI.Core.Events.PointerEventsListener`1[T].Invoke (UI.Core.Events.PointerEventType eventType, UnityEngine.EventSystems.PointerEventData eventData, T item) (at <4029ab2637b6496a9cf4fd9ebd32c4f4>:0)
UI.Core.Radial.Radial`1[T].Invoke (UI.Core.Events.PointerEventType eventType, UnityEngine.EventSystems.PointerEventData eventData, T item) (at <4029ab2637b6496a9cf4fd9ebd32c4f4>:0)
UI.Core.RightClick.RightClickRadialButton.OnPointerClick (UnityEngine.EventSystems.PointerEventData eventData) (at <4029ab2637b6496a9cf4fd9ebd32c4f4>:0)
UnityEngine.EventSystems.ExecuteEvents.Execute (UnityEngine.EventSystems.IPointerClickHandler handler, UnityEngine.EventSystems.BaseEventData eventData) (at <51209b2b252d4c54b13700e32ec0a6f8>:0)
UnityEngine.EventSystems.ExecuteEvents.Execute[T] (UnityEngine.GameObject target, UnityEngine.EventSystems.BaseEventData eventData, UnityEngine.EventSystems.ExecuteEvents+EventFunction`1[T1] functor) (at <51209b2b252d4c54b13700e32ec0a6f8>:0)
UnityEngine.EventSystems.EventSystem:Update()
```
And we have an NRE

**Comment by pyrosshade** [comment]
>SweetExtensions.NetId

Ohohoho, so this wants to be a pain in the ass, eh? Alright.

**Comment by ThatDan123** [comment]
Fixed by #6814
